caom2utils/caom2utils/legacy.py

```diff
@@ -78,7 +80,7 @@
 
 APP_NAME = 'fits2caom2'
 
-__all__ = ['main_app', 'update_blueprint']
+__all__ = ['main_app', 'update_blueprint', 'apply_java_config']
```

Comment:
I'm not sure we need to expose any of this as they are not part of the library API.

Reply:
Removed.